src/isomorphic/React.js

```diff
@@ -109,8 +110,11 @@ if (__DEV__) {
       get() {
         lowPriorityWarning(
           didWarnPropTypesDeprecated,
-          'Accessing PropTypes via the main React package is deprecated. Use ' +
-            'the prop-types package from npm instead.',
+          'In React v16.0+, accessing PropTypes via the main React package is deprecated. Use ' +
```

Comment:
ahhhh yes good catch. sorry I missed that. should say like:

> Accessing PropTypes via the main React package is deprecated. In React v16.0, it will be removed completely. Use the …